modules/katello_repository_set.py

```diff
@@ -132,11 +151,12 @@ def get_desired_repos(desired_substitutions, available_repos):
     return desired_repos
 
 
-def repository_set(module, name, organization, product, state, repositories=[]):
+def repository_set(module, name, organization, product, label, state, repositories=[]):
     changed = False
     organization = find_organization(module, organization)
-    product = find_product(module, product, organization)
```

Comment:
Looking at this old line, I'm fairly sure that never worked.. I'd vote for merging this as is.. If we want we can support that feature in the future :-)